src/ucp/core/ucp_worker.c

```diff
@@ -435,18 +428,8 @@ ucp_worker_iface_error_handler(void *arg, uct_ep_h uct_ep, ucs_status_t status)
     ucp_ep->flags    |= UCP_EP_FLAG_FAILED;
     ucp_ep->am_lane   = 0;
 
-    ucp_ep_errh_iter = kh_get(ucp_ep_errh_hash, &worker->ep_errh_hash,
-                              (uintptr_t)ucp_ep);
-    if (ucp_ep_errh_iter == kh_end(&worker->ep_errh_hash)) {
-        ucs_error("Error %s was not handled for ep %p - "
-                  UCT_TL_RESOURCE_DESC_FMT,
-                  ucs_status_string(status), ucp_ep,
-                  UCT_TL_RESOURCE_DESC_ARG(tl_rsc));
-        return status;
-    }
-
-    err_cb = kh_val(&worker->ep_errh_hash, ucp_ep_errh_iter);
-    err_cb(ucp_ep->user_data, ucp_ep, status);
+    ucp_ep_ext_gen(ucp_ep)->err_cb(ucp_ep_ext_gen(ucp_ep)->user_data, ucp_ep,
```

Comment:
Seems like `err_cb` can be uninitialized here if user doesn't pass UCP_EP_PARAM_FIELD_ERR_HANDLER

Reply:
good catch, fixed